action/remove.go

```diff
@@ -2,24 +2,70 @@ package action
 
 import (
 	"os"
+	"os/exec"
 	"path/filepath"
+	"strings"
 
 	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
 )
 
+// kubeGet executes the kubectl command, override in tests
+var kubeGet func(string) string
```

Comment:
fair point. I may be sliding too far into pedantry at this point, global way is ok with me too. I'm still +1 on the custom type, just as `net/http` does as you said.